src/daos-deserialize/daos-deserialize.c

```diff
@@ -244,7 +371,7 @@ int main(int argc, char** argv)
 
     rc = daos_fini();
     if (rc != 0) {
-        rc = 1;
+        MFU_LOG(MFU_LOG_ERR, "Failed to finalize DAOS (%d)", rc);
```

Comment:
@dsikich Formatted -DER
```suggestion
        MFU_LOG(MFU_LOG_ERR, "Failed to finalize DAOS "DF_RC, DP_RC(rc));
```